modules/pel/peltool/peltool.py

```diff
@@ -563,6 +570,8 @@ def main():
                         dest='IDToDelete', help='Delete a PEL based on its ID')
     parser.add_argument('-D', '--delete-all', dest='deleteAll',
                         action='store_true', help='Delete all PELs')
+    parser.add_argument('-S', '--severity', help='Filter by severity value, consider pels matching the provided value only',
```

Comment:
Though I didn't think of this before, I think when people would use this, they'd want  to see 'all unrecoverable' PELs for example.  However, that is actually 5 different severities:

```
0x40 = Unrecoverable error, general
0x41 = Unrecoverable Error, bypassed with degraded performance
0x44 = Unrecoverable Error, bypassed with loss of redundancy
0x45 = Unrecoverable Error, bypassed with loss of redundancy + perf 
0x48 = Unrecoverable Error, bypassed with loss of function
```

So to make that easier, how about we just do it based on the families:
- nonError  (or informational since that is usually what people say)
- recovered
- predictive
- unrecoverable
- critical
- diagnostic
- symptom

Then you could map those names to just the first nibbles, like 0,1,2,4,5,6,7 and match on that.

Reply:
Agreed, this adds value for the user by eliminating the need to remember the severity list when retrieving PELs.